05-geometry-operations.Rmd

```diff
@@ -22,23 +22,23 @@ After reading it --- and attempting the exercises at the end --- you should unde
 Section \@ref(geo-vec) covers transforming vector geometries with 'unary' and 'binary' operations.
 <!-- TODO: add something on n-ary ops (RL) -->
 Unary operations work on a single geometry in isolation.
-This includes simplification (of lines and polygons), the creation of buffers and centroids, and shifting/scaling/rotating single geometries using 'affine transformations' (sections \@ref(simplification) to \@ref(affine-transformations)).
+This includes simplification (of lines and polygons), the creation of buffers and centroids, and shifting/scaling/rotating single geometries using 'affine transformations' (Sections \@ref(simplification) to \@ref(affine-transformations)).
```

Comment:
I already changed sections to Sections in all chapters in the book. General rules can be implemented on the who book with a tool like regexxer. But definitely no harm in this.

Reply:
ah, good to know! Still, worth to have a look in each chapter because sometimes we have forgotten to write Section and just written `\@ref(...)`